rest/pagination/README.md

```diff
@@ -67,15 +67,12 @@ TODO
 # Using paginated fields
 
 The easiest way to use paginated fields is with a GraphQL operation
-that has variables for the `first` and `after` arguments. Examples
+that has variables for the arguments `first` and `after`. Examples
 are show in `requests.graphql`.
 
-For the first call (to get the first page) the operation is called
-with a value of `first` to set the number of nodes in a page
-and either an empty string (`""`) for `after` or the `after` variable is omitted.
+To retrieve the first page of a result, the operation is called with `after` omitted or set to the empty string (`""`), and `first`, and for all calls, is set to the number of results to return in a page.
```

Comment:
```suggestion
To retrieve the first page of a result, the operation is called with `after` omitted or set to the empty string (`""`), and for all calls, `first` is set to the number of results to return.
```

Reply:
to be clearer, separated out first for subsequent requests.